recipe/build_mamba.bat

```diff
@@ -0,0 +1,42 @@
+if (%PKG_NAME% == "mamba") (
+	cd mamba
+	%PYTHON% -m pip install . --no-deps -vv
+	if (exist %PREFIX%\condabin) (
+		echo "Adding link to mamba into condabin";
+		mklink %PREFIX%/condabin/mamba %LIBRARY_PREFIX%/bin/mamba
```

Comment:
did you compare that to the previous recipe? I think we only need that on Unix :)